rskj-core/src/main/java/co/rsk/mine/MinerClientImpl.java

```diff
@@ -77,7 +77,7 @@ public RefreshWork createRefreshWork() {
     }
 
     public Thread createDoWorkThread() {
-        return new Thread() {
+        return new Thread("miner client") {
```

Comment:
AFAIR, this is an auxiliary class, which its only real use is in regtest. Maybe we could add also the name of the thread to MinerServer, it has a thread

Reply:
Yes, but still, it's nice to have the logs in regtest. I didn't add the name to the MinerSever thread... because the name is already set: https://github.com/rsksmart/rskj/blob/master/rskj-core/src/main/java/co/rsk/mine/MinerServerImpl.java#L194